test/officerManager.js

```diff
@@ -0,0 +1,455 @@
+        })
+    })
+
+    it('Should return statusCode 400 when the required fields are not set', done => {
+      const form = { token: token }
+      chai
+        .request(app)
+        .post('/api/officerManager/submit')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.have.status(400)
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+
+    it('Should return statusCode 500 when no token is submited', done => {
+      const form = {
+        name: 'pinkUnicorn',
+        email: 'test@test.com',
+        level: 2,
+        team: 'dev',
+        major: 'major'
+      }
+      chai
+        .request(app)
+        .post('/api/officerManager/submit')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.have.status(500)
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+
+    it('Should return statusCode 401 when invalid token is submited', done => {
+      const form = {
+        name: 'pinkUnicorn',
+        email: 'test@test.com',
+        level: 2,
+        team: 'dev',
+        major: 'major',
+        token: 'Invalid-Token'
+      }
+      chai
+        .request(app)
+        .post('/api/officerManager/submit')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.have.status(401)
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+
+    it('Should return statusCode 200 when all required fields are filled in', done => {
+      const form = {
+        name: 'pinkUnicorn',
+        email: 'test@test.com',
+        level: 2,
+        team: 'dev',
+        major: 'major',
+        token: token
+      }
+      chai
+        .request(app)
+        .post('/api/officerManager/submit')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.have.status(200)
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+  })
+
+  describe('/POST GetForm', () => {
+    it('Should return statusCode 500 when no token is submited', done => {
+      const form = {
+        email: 'test@test.com'
+      }
+      chai
+        .request(app)
+        .post('/api/officerManager/GetForm')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.have.status(500)
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+
+    it('Should return statusCode 401 when invalid token is submited', done => {
+      const form = {
+        email: 'test@test.com',
+        token: 'Invalid-Token'
+      }
+      chai
+        .request(app)
+        .post('/api/officerManager/GetForm')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.have.status(401)
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+
+    it('Should return an object of all forms', done => {
```

Comment:
i would also add to the test case that the database returns the object you previously added. see [here](https://github.com/SCE-Development/Core-v4/blob/7eb49330115efe0619c97377f7b69382cca2d41d/test/Event.js#L148)